src/ZODB/tests/testSerialize.py

```diff
@@ -139,7 +164,17 @@ def test_funky_datetime_serialization(self):
         # buildout doesn't arrange for the sys.path to be exported,
         # so force it ourselves
         environ = os.environ.copy()
-        environ['PYTHONPATH'] = os.pathsep.join(sys.path)
+        if IS_JYTHON:
+            # Jython 2.7rc2 has a bug; if its Lib directory is
+            # specifically put on the PYTHONPATH, then it doesn't add
+            # it itself, which means it fails to 'import site' because
+            # it can't import '_jythonlib' and the whole process fails
+            # We would use multiprocessing here, but it doesn't exist on jython
+            sys_path = [x for x in sys.path
+                        if not x.endswith('Lib') and x != '__classpath__' and x!= '__pyclasspath__/']
```

Comment:
I just tried to create a small test case to file a bug report with, and I can't reproduce this problem in Jython 2.7.0. It's definitely there in 2.7rc2, although I don't see any specific release notes about fixing something like this for 2.7.0. It's possible it was in the meta-bug [2319](http://bugs.jython.org/issue2319) which was fixed for rc3.